internal/service/listeners/deploy.go

```diff
@@ -1,10 +1,12 @@
 package listeners
 
 import (
-	"fmt"
-
+	"github.com/dov-id/CertIntegrator-svc/internal/data"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	sql "github.com/iden3/go-merkletree-sql/db/pgx/v2"
+	"github.com/iden3/go-merkletree-sql/v2"
+	"gitlab.com/distributed_lab/logan/v3/errors"
 )
 
 type LogDeploy struct {
```

Comment:
Use abigen for the go code generation from the contract code. It provides you go structures of the events and convenient methods for parsing them from the go-ethereum/core/types.Log